content/03-type-system.md

```diff
@@ -315,9 +315,9 @@ Given a set of multiple types, a **common base type** is a type which all types
 
 Although `Base` is not mentioned, the Haxe Compiler manages to infer it as the common type of `Child1` and `Child2`. The Haxe Compiler employs this kind of unification in the following situations:
 
-* array declarations
-* `if`/`else`
-* cases of a `switch`
+* Array declarations.
+* `if`/`else`.
+* Cases of a `switch`.
```

Comment:
Are these the only cases? How about the ternary operator?

Reply:
I don't mind adding anything else that should be added here, as long as you can give me a full list of what's needed. 

To be clear however, I'm not attempting to review or verify the accuracy of anything in the manual; that's for someone else to handle.

Reply:
I understand, I am not aiming to make this PR be the ultimate correction of this file. Seeing it just made me wonder, and the question is primarily for @Simn or @RealyUniqueName.

Reply:
Also for `try...catch...catch...`